bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -45,11 +49,38 @@
   /** Constant <code>LOG</code> */
   protected static final Logger LOG = new Logger(ChannelPool.class);
 
+  private static final AtomicInteger ChannelIdGenerator = new AtomicInteger();
+
+  /**
+   * Best effort counter of active channels. There may be some cases where channel termination
+   * counting may not accurately be decremented.
+   */
+  protected static Counter ACTIVE_CHANNEL_COUNTER;
+
+  protected synchronized static Counter getActiveChannelCounter() {
+    if (ACTIVE_CHANNEL_COUNTER == null) {
```

Comment:
Curious, why not just initialize in the ChannelPool constructor rather than lazily? 

Is it because you want to avoid the initialization in the case where the metrics stuff is turned off?